src/MongoDB/BulkWriteCommand.stub.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @generate-class-entries static
+ * @generate-function-entries static
+ */
+
+namespace MongoDB\Driver;
+
+/** @not-serializable */
+final class BulkWriteCommand implements \Countable
+{
+    final public function __construct(?array $options = null) {}
+
+    // TODO: decide if Countable is still desirable
+    public function count(): int {}
```

Comment:
I don't see the use-case for counting. Is it the number of operations it contains? Maybe only to know if it's empty.

Reply:
It would be the number of appended operations. I'm not convinced this is necessary, and it would be trivial to add at a later point if some user requested it (presumably because they relied on it on the original BulkWrite class).